book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] As the previous reviewer has earlier suggested, a Rental belongs to both the Book class and a Person class, therefore we need to ensure that when creating a new rental, we create the belong-to relationship between the above-mentioned classes.

You have to create a new instance of a Rentals with the Person, Book, and date parameters to establish the many-to-many relationship
e.g 
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

What you implemented would have been correct if you added the actual rental parameters you are passing to the add_rental method

e.g 

```
  def add_rental(date, person)
    @rentals.push(rental) unless @rentals.include?(rental)
    rental.book = self
  end
```